command/genesis/params.go

```diff
@@ -23,6 +24,8 @@ const (
 	epochSizeFlag           = "epoch-size"
 	blockGasLimitFlag       = "block-gas-limit"
 	posFlag                 = "pos"
+	minNumValidators        = "min-validator-count"
+	maxNumValidators        = "max-validator-count"
```

Comment:
Flags are called `<>-validator-count`, but the flag references are called `<>NumValidators`

Please consider keeping the naming in check, like `minValidatorCount` and `maxValidatorCount`